data/afrique/fix/index.json

```diff
@@ -0,0 +1,3398 @@
+{
+  "metadata": {
+    "id": "urn:bbc:ares::index:afrique/48465371/desktop/domestic",
+    "locators": {
+      "assetUri": "/afrique/48465371",
```

Comment:
Should this file not be called `48465371.json`? A service can have multiple, unlike frontpages which has 1, so `index` is fine.

Reply:
Ah ok, that makes sense. Thanks!